cli/pkg/kctrl/cmd/app/init/template.go

```diff
@@ -103,8 +104,29 @@ func (t *TemplateStep) Interact() error {
 	return nil
 }
 
+func (t *TemplateStep) getHelmAppTemplate(fetchSource string) (v1alpha1.AppTemplate, error) {
+	var pathFromVendir string
+	if fetchSource == FetchChartFromGit {
+		vendirConf, err := ReadVendirConfig()
+		if err != nil {
+			return v1alpha1.AppTemplate{}, err
+		}
+		pathFromVendir = vendirConf.Directories[0].Contents[0].IncludePaths[0]
+		// TODO Create regex and match against that.
```

Comment:
I think this is good enough if there are  no additional possibilities. Are we sure about this?

Is "/**/*/" valid? Should that be taken care of? (I would think not but just making sure)

Reply:
I am removing all the trailing `/` before using TrimSuffix to handle n no. of trailing `/`. e.g. `/**/*////`,

Reply:
Adopted